web/src/lib/components/asset-viewer/detail-panel-star-rating.svelte

```diff
@@ -0,0 +1,98 @@
+<script lang="ts">
+  import { handleError } from '$lib/utils/handle-error';
+  import { updateAsset, type AssetResponseDto } from '@immich/sdk';
+  import { t } from 'svelte-i18n';
+
+  export let asset: AssetResponseDto;
+  export let isOwner: boolean;
+
+  const countStars = 5;
+
+  $: rating = asset.exifInfo?.rating || 0;
+  let currentRating = rating;
+
+  const handleChangeRating = async () => {
+    if (currentRating === asset.exifInfo?.rating) {
+      return;
+    }
+    try {
+      await updateAsset({ id: asset.id, updateAssetDto: { rating: currentRating } });
+    } catch (error) {
+      handleError(error, $t('cant_apply_changes'));
```

Comment:
Message doesn't exist

```suggestion
      handleError(error, $t('errors.cant_apply_changes'));
```